src/_includes/docs/swift-package-manager/migrate-objective-c-plugin.md

```diff
@@ -0,0 +1,372 @@
+
+   <pre>
+   plugin_name/plugin_name_ios/ios/
+   ├── ...
+   └── <b>plugin_name_ios/</b>
+   </pre>
+
+3. Within this new directory, create the following files/directories:
+
+    - `Package.swift` (file)
+    - `Sources` (directory)
+    - `Sources/plugin_name_ios` (directory)
+    - `Sources/plugin_name_ios/include` (directory)
+    - `Sources/plugin_name_ios/include/plugin_name_ios` (directory)
+    - `Sources/plugin_name_ios/include/plugin_name_ios/.gitkeep` (file)
+      - This file ensures the directory is committed.
+        You can remove the `.gitkeep` file if other files are added to the
+        directory.
+
+   Your plugin should look like:
+
+   <pre>
+   plugin_name/plugin_name_ios/ios/
+   ├── ...
+   └── plugin_name_ios/
+      ├── <b>Package.swift</b>
+      └── <b>Sources/plugin_name_ios/include/plugin_name_ios/</b>
+         └── <b>.gitkeep</b>
+   </pre>
+
+4. Use the following template in the `Package.swift` file:
+
+   ```swift title="Package.swift"
+   // swift-tools-version: 5.9
+   // The swift-tools-version declares the minimum version of Swift required to build this package.
+   
+   import PackageDescription
+   
+   let package = Package(
+       name: "plugin_name_ios",
+       platforms: [
+           // The platforms your plugin supports.
+           // If your plugin only supports iOS, remove `.macOS(...)`.
+           // If your plugin only supports macOS, remove `.iOS(...)`.
+           .iOS("12.0"),
+           .macOS("10.14")
+       ],
+       products: [
+           // If the plugin name contains "_", replace with "-" for the library name
+           .library(name: "plugin-name-ios", targets: ["plugin_name_ios"])
+       ],
+       dependencies: [],
+       targets: [
+           .target(
+               name: "plugin_name_ios",
+               dependencies: [],
+               resources: [
+                   // If your plugin requires a privacy manifest, for example if it uses any required
+                   // reason APIs, update the PrivacyInfo.xcprivacy file to describe your plugin's
+                   // privacy impact, and then uncomment these lines. For more information, see
+                   // https://developer.apple.com/documentation/bundleresources/privacy_manifest_files
+                   // .process("PrivacyInfo.xcprivacy"),
+   
+                   // If you have other resources that need to be bundled with your plugin, refer to
+                   // the following instructions to add them:
+                   // https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package
+               ],
+               cSettings: [
+                   .headerSearchPath("include/plugin_name_ios")
+               ]
+           )
+       ]
+   )
+   ```
+
+   :::note
+   If the plugin name contains `_`, the library name must be a `-` separated
+   version of the plugin name.
+   :::
+
+5. If your plugin has a `PrivacyInfo.xcprivacy` file, move it to
```

Comment:
Link to Apple docs https://developer.apple.com/documentation/bundleresources/privacy_manifest_files ?

Reply:
Good idea, added!